### DISCUSSION TITLE
Is there plans to support Craft 4's new address element?

### DISCUSSION THREAD

**Discussion by john-henry** [engram-design/FeedMe#1145] [General]
Is there plans to support Craft 4's new address element where I can import Users and their Addresses?

**Comment by john-henry** [comment]
Realize this should likely be in Discussions?

**Comment by angrybrad** [comment]
Plans, yeah. No real timeframe for it at the moment, though.

**Reply by nilsenpaul** [reply]
Hi Brad! Any update on this? Would be nice to be able to import one or more addresses for a user when importing users into Commerce!

**Reply by angrybrad** [reply]
@nilsenpaul sorry, no update on timeframe. :-/

**Comment by peteeveleigh** [comment]
Any chance of this happening in the next 12 hours? 🫣

**Reply by KevinBodi** [reply]
It looks like it's now supported in the newest version https://github.com/craftcms/feed-me/blob/6.x/CHANGELOG.md#6100---2025-08-15

**Reply by angrybrad** [reply]
12 hours and 3 months? :P

**Comment by chasegiunta** [comment]
@angrybrad I didn't have any luck with it 😔 ... Fields look as if they are mapped fine when setting up the import, and the owner entries are imported perfectly, but the addresses remain empty. The feedme logs don't show any errors (even if, most likely, there was validation errors), simply with `address: []` in the log output.

**Reply by angrybrad** [reply]
Can you shoot a db dump, composer.json/lock files and a sample feed where we can reproduce the issue over to support@craftcms.com?
